release-notes/opensearch-security-dashboards-plugin.release-notes-2.4.0.0.md

```diff
@@ -0,0 +1,26 @@
+## 2022-11-10 Version 2.4.0.0
+
+Compatible with Opensearch-Dashboards 2.4.0
+
+### Features
+* Initial commit for multiple authentication ([1110](https://github.com/opensearch-project/security-dashboards-plugin/pull/1110))
```

Comment:
Can you add `#` in front of the issue number to keep this consistent with other release notes?. 

i.e. 

[#1110] instead of [1110]

Reply:
Thanks for the reminder! I just fixed.